streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1154,11 +1152,7 @@ protected StandbyTask createStandbyTask(final TaskId id, final Collection<TopicP
         final ProcessorTopology topology = builder.build(id.topicGroupId);
 
         if (!topology.stateStores().isEmpty()) {
-            try {
-                return new StandbyTask(id, applicationId, partitions, topology, consumer, storeChangelogReader, config, streamsMetrics, stateDirectory);
-            } finally {
-                log.trace("{} Created standby task {} with assigned partitions {}", logPrefix, id, partitions);
```

Comment:
As above.

Reply:
We do not need to, as the returned value would be printed again in the `addNewTask` call (that's also why I added a few more pieces of info there).